app/validators/work_subtype_validator.rb

```diff
@@ -13,12 +13,12 @@ def validate_each(record, attribute, value)
   # validate these on the incoming works/new request before there is a model
   # instance to validate.
   def self.valid?(work_type, value)
+    # A subtype is required for the "other" work type
     return Array(value).first.present? if work_type == 'other'
 
-    # Subtype is not required for work types other than 'other'
     return true if value.nil?
 
-    value.all? { |subtype| subtype.in?(WorkType.subtypes_for(work_type)) }
+    value.all? { |subtype| subtype.in?(WorkType.subtypes_for(work_type, include_more_types: true)) }
```

Comment:
Amy has confirmed that loosening validation of subtypes is A-OK from her perspective and from metadata's.